app/src/main/res/values/strings.xml

```diff
@@ -74,5 +74,16 @@
         <item>Remove ads forever</item>
         <item>Remove ads temporarily for free by watching a short video</item>
     </string-array>
+    
+    <!--  iOS -->
+    <string name="ok">OK</string>
+    <string name="unsaved_changes">You have unsaved changes</string>
+    <string name="save_now">Save them now?</string>
+    <string name="yes">Yes</string>
+    <string name="discard_changes">Discard changes</string>
+    <string name="cancel">Cancel</string>
+    <string name="enter_password">Please enter the password to decrypt this document</string>
```

Comment:
we said we can replace that with toast_error_password_protected on iOS, no?

Reply:
that is correct